Cargo.lock

```diff
@@ -395,6 +412,12 @@ dependencies = [
  "loom",
 ]
 
+[[package]]
+name = "bytes"
+version = "1.0.0"
```

Comment:
might be a good idea to also check/update all other dependencies. since this now includes `0.4`, `0.5` and `1`

Reply:
this comes from multiple tokio versions which comes from multiple hyper versions which comes from multiple reqwest versions

we can update sentry sdk to fix this but I would do that in a separate PR

Reply:
We can't, since we mostly have to deal with the `bytes` we get from actix, which is still on `0.4`. We can only resolve this once we upgrade the web layer.